working-with-your-data/working-with-your-own-data.md

```diff
@@ -114,6 +118,63 @@ wget --user=<USERNAME> --ask-password <URL>
 ```
 Using the `--ask-password` will prompt you to enter your password.
 
+### Upload large files (> 1Gb) from your own computer
+
+#### Install FileZilla
+
+Go to [FileZilla's website](https://filezilla-project.org/download.php?type=client) to download the FileZilla Client for your respective operating system.
```

Comment:
Taking notes as I go here - when I follow this link and it detects (presumably) my OS as Linux 64bit, that page says

> Built for Debian 10.0 (Buster) 64bit edition. It is highly recommended to use the package management system of your distribution or to manually compile FileZilla if you are running a different flavour of Linux.

So one can use Ubuntu Software Center (for Ubuntu, but that's the only thing I'm testing) or the command line ([ref](https://www.computernetworkingnotes.com/linux-tutorials/how-to-install-filezilla-ftp-client-in-ubuntu.html)).